### ISSUE TITLE
[1.12] Crash with Forestry

### ISSUE THREAD

**Issue by MasterEnderman** [Elecs-Mods/ElecCore#135]
Forge: 14.23.2.2654
ElecCore: 1.8.434
Forestry: 5.8.0.263
Crash-Report: https://pastebin.com/dVjRzD1v

**Comment by Matakor** [comment]
I'm getting this same crash. No indication as to actual cause, outside crash report.

**Comment by Elec332** [comment]
Can you reproduce this with the latest version of forestry? Version 5.8.0.290

**Comment by Matakor** [comment]
Probably not, no. The crash happened only once, then did not happen again. (been playing since with no issues)

It seems to only happen on a new install.

**Comment by Elec332** [comment]
Ill leave this open for another week, and try to reproduce it.
If anyone can find a way to reproduce it with forestry 5.8.0.290 or later, please tell me.

**Comment by Elec332** [comment]
I've had no further reports of this issue, please make a new issue if the problem arises again (with Forestry  5.8.0.290 or later)
